client/web/src/components/login-toggle.tsx

```diff
@@ -192,6 +199,15 @@ function LoginPopoverContent({
                   <p className="text-gray-500 text-xs">
                     You can see most of this device's details. To make changes,
                     you need to sign in.
+                    {isHTTPS && (
+                      // we don't know if the user can connect over TS, so
+                      // provide extra tips in case they have trouble.
+                      <>
+                        {" "}
+                        Make sure you are connected to your tailnet, and that
+                        your policy file allows access.
+                      </>
```

Comment:
Maybe we put this in a second paragraph, so it's a little harder to miss. We could even play with tweaking the font a tad.

Reply:
done.  New screenshot:
<img width="503" alt="Screenshot 2024-01-11 at 2 30 53 PM" src="https://github.com/tailscale/tailscale/assets/1112/e8b91d3a-bee2-4f72-83f1-5c31786d24e2">

Reply:
Verrrryyy nit, but maybe throw `font-medium` on it?

Reply:
`font-medium` is almost not even noticeable:
<img width="505" alt="Screenshot 2024-01-11 at 2 40 04 PM" src="https://github.com/tailscale/tailscale/assets/1112/beffbfd9-432d-4f47-a116-ed5c5f1bb2e2">

Maybe `font-semibold` ?
<img width="502" alt="Screenshot 2024-01-11 at 2 40 18 PM" src="https://github.com/tailscale/tailscale/assets/1112/50df97af-8d53-406c-8bfc-74d87064df9d">

Reply:
yeah I like semibold, let's go with that!